x-pack/test/functional/apps/infra/logs_source_configuration.ts

```diff
@@ -74,7 +74,9 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
       });
 
       it('renders the default log columns with their headers', async () => {
-        await logsUi.logStreamPage.navigateTo();
+        await logsUi.logStreamPage.navigateTo({
+          logPosition: { start: '2018-10-17T19:42:22.000Z', end: '2018-10-17T19:57:21.000Z' },
```

Comment:
Would it be worth introducing two constants for these dates to ensure consistency across test cases? There is a `DATES` constant in `./constants` which we might want to extend with this.